src/main/java/seedu/address/logic/parser/FindCommandParser.java

```diff
@@ -20,14 +21,18 @@ public class FindCommandParser implements Parser<FindCommand> {
      */
     public FindCommand parse(String args) throws ParseException {
         String trimmedArgs = args.trim();
-        if (trimmedArgs.isEmpty()) {
+
+        if (trimmedArgs.startsWith(PREFIX_SPECIFIC_FIND.getPrefix()) && !trimmedArgs.isEmpty()) {
```

Comment:
Is this if statement acceptable, do you think?

Reply:
I think all's good

Reply:
Looks ok